core-contracts/FeeDistribution/src/main/java/finance/omm/score/fee/distribution/AbstractFeeDistribution.java

```diff
@@ -52,6 +53,9 @@ protected void distributeFee(BigInteger amount) {
                 accumulatedFee.set(receiver, feeAccumulatedAfterClaim.add(amountToDistribute));
             }
         }
+        if (remaining.signum() > 0){
+            call(Contracts.sICX, "transfer", daoFundAddr, remaining);
```

Comment:
Is this line ok?
In line 48, "amountToDistribute" is added to collectedFee, but the remaining is not.